specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingProfile.json

```diff
@@ -43,7 +43,7 @@
           },
           {
             "name": "$expand",
-            "description": "May be used to expand the invoice sections.",
+            "description": "May be used to get deleted billing profiles or expand the invoice sections.",
```

Comment:
Do we allow someone to permanently delete a billing profile? E.g. if I have a billing profile without any data, is there any reason I would ever see it again after I delete it?

Reply:
When we delete a billing profile, its status is changed to deleted but do not permanently delete it. We will allow users to retrieve the deleted billing profiles. This applied for invoice sections as well.